### DISCUSSION TITLE
Anonymous cant view password protected album (docker version)

### DISCUSSION THREAD

**Discussion by crtvrmn** [LycheeOrg/Lychee#2790] [Bugs] [answered]
### Lychee version

v6.1.2

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

this might be related to the fix from [/Lychee/issues/2675](https://github.com/LycheeOrg/Lychee/issues/2675).

anonymous can not see a password protected album if the password is provided

### Steps to reproduce the issue

**current behaviour** (not logged in)
1. enter url of passwordprotected album
2. enter password of album 
3. get redirected to login _thismightbethecatch_
4. enter credtials (no longer anonymous!)
5. see album (logged in)

**current behaviour** (logged in)
1. enter url of passwordprotected album
2. see album

**expected behaviour ** (not logged in and maybe logged in)
1. enter url of passwordprotected album
2. enter password of album
3. see album

### Diagnostics [REQUIRED]

info
Latest version of PHP is 8.3
info
git (software) is not available.
info
Found 329 placeholders that could be generated.
info
Have you considered supporting Lychee? :)

### Browser & System [REQUIRED]

tested clients:
linux debian12 chrome
windows10 chrome & edge
androidnooneknowstheseversions & chrome

server:
docker version, 6.1.2

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment]
Sorry for the delay, crazy busy here.
That is surprising, I will try to inspect that.

**Reply by ildyria** [reply, marked as answer]
I am not able to reproduce, can you double check with version 6.2.0 ?

**Reply by crtvrmn** [reply]
tested with 6.2.0, works as expected. no need to say sorry for maintaining a foss project!
